CI fail with windows

## Bug description

2023-07-05T10:26:03.8372712Z [libcec-sys 4.0.1-alpha.0] LINK : fatal error LNK1104: cannot open file 'python311_d.lib'

<!-- A clear and concise description of what the bug is. -->

## To Reproduce

<!--
Steps to reproduce the behavior. Something like:
1. Go to '...'
2. Click on '....'
3. Scroll down to '....'
4. See error
-->

## Expected behavior

<!-- A clear and concise description of what you expected to happen. -->

## Screenshots

<!-- If applicable, add screenshots to help explain your problem. -->

## Environment

<!-- Please fill the following information. -->

- OS: [e.g. iOS]
- Browser [e.g. chrome, safari]
- Version [e.g. 22]

## Additional context

<!-- Add any other context about the problem here. -->

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -160,12 +160,13 @@ jobs:
               run: sudo apt-get remove -yq pkg-config
             # install Python with debug binaries for Windows (needed for debug build of vendored libcec)
             # Github Workflow runners do not package debug binaries by default
+            # NOTE: 
             - if: runner.os == 'Windows'
-              name: Install python 3.10.5 with debug binaries
+              name: Install python 3.11.4 with debug binaries
               run: |
                   $arch = '${{ runner.arch }}' -eq 'X86' ? "" : "-amd64"
-                  curl -o python-3.10.5$arch.exe https://www.python.org/ftp/python/3.10.5/python-3.10.5$arch.exe
-                  Start-Process -FilePath "python-3.10.5$arch.exe" -ArgumentList "/quiet Include_debug=1 PrependPath=1 InstallAllUsers=0" -Verb runas -Wait
+                  curl -o python-3.11.4$arch.exe https://www.python.org/ftp/python/3.11.4/python-3.11.4$arch.exe
+                  Start-Process -FilePath "python-3.11.4$arch.exe" -ArgumentList "/quiet Include_debug=1 PrependPath=1 InstallAllUsers=0" -Verb runas -Wait
             - name: Cargo test
               env:
                   EXPECTED_LIBCEC_VERSION_MAJOR: ${{ matrix.job.expected_libcec_abi }}

